spacetemplate/assets/legacy.yaml

```diff
@@ -51,7 +59,9 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Value Proposition
   can_construct: yes
-  description: TBD
+  description: A statement of the value provided to the user by addressing a
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed